netket/optimizer/stochastic_reconfiguration.py

```diff
@@ -199,6 +208,39 @@ def compute_update(self, oks, grad, out=None):
         self._comm.barrier()
         return out
 
+    def _jax_cg_solve(self,oks,grad,n_samp):
+        """
+        Solves the SR flow equation using the conjugate gradient method 
+        """
+
+        n_par = grad.shape[0]
+        if self._x0 is None:
+            self._x0 = _np.zeros(n_par, dtype=_np.complex128)
+
+        cov_op = self._jax_linear_function(oks,n_samp)
```

Comment:
@chrisrothUT have you tried to jit this function to see if you have performance improvements? 
say
```python
cov_op = jax.jit(self._jax_linear_function(oks,n_samp))
```

Reply:
I'll give that a try

Reply:
For some reason that actually makes performance way worse (by a factor of 2/3). Maybe it is compiling every time?

Reply:
yes, that's possible... you should try to jit this only once at construction times, but that might have some side effects if not done properly uhm